css/photography.css

```diff
@@ -66,15 +71,65 @@
     width:100%;
     max-width: 525px;
     height:425px;
+    cursor: pointer;
+    
+  }
+
+
+  .portfolio__grid .portrait p{
+    font-size: 10rem;
+    position:fixed;
+    left:300px;
+    top:300px;
+
+  }
+
+  .portfolio__grid .portrait .portrait-text{
+    display:none;
+  }
+
+  .portfolio__grid .automotive p{
+    font-size: 10rem;
+    position:fixed;
+    top:300px;
+    left:300px;
```

Comment:
Please consolidate these styles so not so much repetition and a singular configurable place